egs2/ml_superb2/asr1/local/score.py

```diff
@@ -0,0 +1,101 @@
+import argparse
+import os
+import re
+import statistics
+import string
+import unicodedata
+
+from jiwer import cer
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--exp_dir")
+args = parser.parse_args()
+
+
+def remove_punctuation(sentence):
+    new_sentence = ""
+    for char in sentence:
+        # all unicode punctuation is of type P
+        if unicodedata.category(char).startswith("P"):
+            continue
+        else:
+            new_sentence = f"{new_sentence}{char}"
+    return new_sentence
+
+
+def normalize_and_calculate_cer(ref, hyp, remove_spaces):
+    if remove_spaces:
+        hyp = re.sub(r"\s", "", hyp)
+        ref = re.sub(r"\s", "", ref)
+
+    # remove punctuation
+    hyp = remove_punctuation(hyp)
+    ref = remove_punctuation(ref)
+
+    hyp = hyp.upper()
+    ref = ref.upper()
+    if len(ref) == 0:
+        return -1
+    return cer(ref, hyp)
+
+
+def calculate_acc(hyps, refs):
+    acc = 0
+    for hyp, ref in zip(hyps, refs):
+        if hyp == ref:
+            acc += 1
+    return acc / (len(refs))
+
+
+def score(references, lids, hyps):
```

Comment:
Can you add an example of how the sentence is changed before/after the normalization for some typical languages as a comment?

Ditto in the webpage.

If we have some normalization, please **_always_** show an example of how the normalization is conducted.